cvat-ui/src/cvat-store.ts

```diff
@@ -21,9 +22,10 @@ const middlewares = [
 let store: Store | null = null;
 
 export default function createCVATStore(createRootReducer: () => Reducer): void {
+    // todo enable devtools only in dev mode
```

Comment:
So, for now does it work in production?
Do you happen to know how it affects performance?

Reply:
It's easier to remove todo:)